gitlab2rbac.py

```diff
@@ -159,9 +285,20 @@ def get_users(self, from_namespaces=None):
     def get_groups(self):
         groups = []
         for group in self.groups:
-            for result in self.client.groups.list(search=group, all=True):
-                if result.parent_id is None:
-                    logging.info(u"|found group={}".format(result.name))
+            _start = time()
+            gitlab_groups = self.client.groups.list(
+                search=group,
+                top_level_only=True,
+                all=True,
+            )
+            timespent = time() - _start
+            logging.debug(f"Fetched groups in {timespent} seconds")
+            for result in gitlab_groups:
+                if (
+                    result.parent_id is None
```

Comment:
Isn't this redundant with `top_level_only=True` ?

Reply:
Oh yes, that's something I forgot to remove when I found out I could filter the parent group only
I'm gonna test it without